.babelrc

```diff
@@ -1,11 +1,4 @@
 {
-  "presets": [
-    ["env"],
-    "react"
-  ],
-  "plugins": [
-    "transform-object-rest-spread",
-    "transform-class-properties",
-    "transform-export-extensions"
-  ]
+  "presets": [["env", { "modules": false }], "stage-2", "react"],
+  "plugins": ["transform-class-properties"]
```

Comment:
Here you don't need to use the [`stage-2`](https://babeljs.io/docs/plugins/preset-stage-2/)
Don't you still need `transform-object-rest-spread` ?

Also I used `transform-export-extensions` **without** `{ "modules": false }` to be able to transform `export Manager from './Manager'` correctly. But it's a problem with rollup if you want to use his native support of esm.
We could use babel env feature to use rollup native modules & let babel transform for the lib build

Reply:
@FezVrasta 
Do we need everything in stage-2 & stage-3 ?
I think we only need `transform-class-properties` and `transform-object-rest-spread`, the `babel-preset-env` takes care of the rest.
Anyway it's ok to use stage-2 but `transform-class-properties` becomes redundant here

Reply:
I kept `stage-2` and removed the plugin